README.md

```diff
@@ -1,19 +1,32 @@
-### Hello there :suspect:, I am Paul Artushkov (also known as pavTiger)
-
-Since the early childhood, I've been interested in technology. I started with soldering some boards and Arduino components. 
+## Hello there :suspect:, I am Paul Artushkov (also known as pavTiger)
+Since early childhood, I've been interested in technology. I started with soldering some boards and Arduino components. 
 Then I slowly switched to Robotics olympiads with LEGO EV3 system. And went to RRO (Russian Robot Olympiad) in <b>Innopolis</b>. 
 As I went to the fifth grade, I switched to more practical programming (that was the time when I started using <b>GitHub</b>). 
 Enjoy hackathons very much and have been to many of <b>CROC<b> hacks (and won a few). Recently, I started coding Olympiad tasks in <b>Tinkoff Generation<b>.
 
 I study in an IT & Physics school named Silaeder, where we do many projects and hackathons.
 
-My biggest projects are:
-* Physarum modeling
-* TRON Legacy game
+Good-known languages: Python, C++, Javascript, Pascal
+English level B2 proven by Cambridge exam PET (Currently my level is ~C1, but I didn't take the exam yet)
+
+## My biggest projects are:
+* [Physarum modeling](https://github.com/physarumAdv)
+* [TRON Legacy game](https://github.com/light-merch/TRON_multiplayer)
+* [Moscow Metro analytics](https://github.com/pavtiger/Moscow-Underground-Analysus)
 * Chatbots
 * And many more hackathons
 
-Recently I went to an Internship to a company named Visyond.
+Just before COVID-19, I went to Saint Petersburg to participate in [Balt Konkurs](https://baltkonkurs.ru). Won 3rd-degree award and Youth jury laureate prize in "Computer Science'.
+Recently I went on an Internship at a company named Visyond.
 They are doing an excel for business people. It was a great experience☺️
```

Comment:
Not sure the word "doing" fits well enough, but not either sure which word would be perfect